juntagrico/views_subscription.py

```diff
@@ -306,11 +307,23 @@ def activate_subscription(request, subscription_id):
     change_date = request.session.get('changedate', None)
     try:
         subscription.activate(change_date)
+        add_subscription_member_to_activity_area(subscription)
     except ValidationError as e:
         return error_page(request, e.message)
     return return_to_previous_location(request)
 
 
+def add_subscription_member_to_activity_area(subscription):
+    activity_area = ActivityArea.objects.filter(auto_add_new_members=True)
+    member_ids = SubscriptionMembership.objects.filter(subscription=subscription).values_list('member')
```

Comment:
There is a property for the recepients in the subscription, which should be used here

Reply:
done